spec/support/matchers/have_value.rb

```diff
@@ -0,0 +1,15 @@
+RSpec::Matchers.define :have_value do |expected|
+  match do |variable|
+    selector_id = variable.sub('$', '#')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.